content/en/serverless/installation/nodejs.md

```diff
@@ -278,6 +280,15 @@ exports.handler = async (event) => {
   return response;
 };
 ```
+Learn more about submitting custom metrics [here][3].
+
+### Enable the AWS X-Ray Integration
+
+Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].
```

Comment:
Do the APM docs talk about "trace data"? The bigger picture here is visualizing entire end-to-end requests to serverless functions